workspace/notebook/test_smoke_py_connectivity.json

```diff
@@ -116,9 +116,8 @@
 					"        print(exception)\n",
 					"        print()\n",
 					"    \n",
-					"    failing_tests = list(exception_map.keys())\n",
-					"    mssparkutils.notebook.exit(failing_tests)\n",
-					"mssparkutils.notebook.exit([])"
+					"    mssparkutils.notebook.exit(exception_map)\n",
+					"mssparkutils.notebook.exit(dict())"
```

Comment:
You're having two exits here ? only the last one run so you will always exit with an empty dict and mask the real failure